app/model/editions/templates/TheDummyEdition.scala

```diff
@@ -0,0 +1,200 @@
+package model.editions.templates
+
+import java.time.ZoneId
+
+import model.editions.Swatch._
+import model.editions._
+import model.editions.templates.TemplateHelpers._
+
+//noinspection TypeAnnotation
+object TheDummyEdition {
+  lazy val template = EditionTemplate(
+    List(
+      FrontSpecial01 -> Daily(),
```

Comment:
can we have more variations in `TheDummyEdition` on how we can set time window

Reply:
They are present in the Training Edition - the Dummy Edition is for David to do mockups in.